easybuild/easyblocks/g/gurobi.py

```diff
@@ -43,13 +44,24 @@ class EB_Gurobi(Tarball):
 
     def install_step(self):
         """Install Gurobi and license file."""
-        # make sure license file is available
-        if self.cfg['license_file'] is None or not os.path.exists(self.cfg['license_file']):
-            raise EasyBuildError("No existing license file specified: %s", self.cfg['license_file'])
+
+        # Check if the client license file is already defined in the appropriate easyconfig variable
+        licfile = self.cfg['license_file']
+        if licfile is None:
+            # Fallback to standard Gurobi license file environment variable
+            licfile = os.getenv('GRB_LICENSE_FILE')
```

Comment:
Have you considered using the `find_flexlm_license` function provided by `filetools`? See https://github.com/easybuilders/easybuild-framework/blob/develop/easybuild/tools/filetools.py#L1670

See the COMSOL & PGI easyblocks, as well as the `IntelBase` generic easyblock used for the Intel tools.